src/bot.ts

```diff
@@ -14,10 +13,10 @@ const spotify = new Spotify({
   refreshToken: process.env.SPOTIFY_REFRESH_TOKEN as string
 })
 
-// INFO: if you don't need lastfm info, comment this
 const lastfm = new Lastfm({
   key: process.env.LASTFM_API_KEY as string
 })
+const isLastFmUsing = process.env.LASTFM_API_KEY?.length
```

Comment:
`is` at the start of the variable name refers to a variable as a type of `boolean`, but actually `number` is used. i would consider doing this then:
```js
const isUsingLastfm = process.env.LASTFM_API_KEY !== undefined
```

Reply:
changed with last commit, now uses `&&`, that returns `boolean` in final

Reply:
> changed with last commit, now uses `&&`, that returns `boolean` in final

i don't think so!

```js
const foo = 'foo'.length && 'bar'.length
console.log(foo) // 3
```